docs/standard/frameworks.md

```diff
@@ -14,54 +14,31 @@ ms.assetid: 6ef56a2e-593d-497b-925a-1e25bb6df2e6
 
-You can refer to a .NET platform or runtime. Both are equally valid.
-
-* .NET Framework 4.6.2
-* .NET 4.6.2
-
-**Referring to a framework**
-
-You can refer to a framework or targeting of a framework using long- or short-forms of the Target Framework Moniker (TFM). Both are equally valid.
-
-* `.NETFramework,Version=4.6.2`
-* `net462`
-
-**Referring to a family of frameworks**
-
-You can refer to a family of frameworks using long or short-forms of the framework ID. Both are equally valid.
-
-* `.NETFramework`
-* `net`
-
-## Latest framework versions
-
-The following table defines the set of frameworks that you can use, how they're referenced, and which version of the [.NET Standard](net-standard.md) they implement. These framework versions are the latest stable versions. Pre-release versions aren't shown.
-
-| Framework             | Latest Version | Target Framework Moniker (TFM) | Compact Target Framework Moniker (TFM) | .NET Standard Version | Metapackage |
+| Target Framework      | Latest Version | Target Framework Moniker (TFM) | Compact Target Framework Moniker (TFM) | .NET Standard Version | Metapackage |
 | :-------------------: | :------------: | :----------------------------: | :------------------------------------: | :-------------------: | :---------: |
 | .NET Standard         | 1.6.1          | .NETStandard,Version=1.6       | netstandard1.6                         | N/A                   | [NETStandard.Library](https://www.nuget.org/packages/NETStandard.Library) |
-| .NET Core Application | 1.1.1          | .NETCoreApp,Version=1.1        | netcoreapp1.1                          | 1.6                   | [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App) |
+| .NET Core Application | 1.1.2          | .NETCoreApp,Version=1.1        | netcoreapp1.1                          | 1.6                   | [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App) |
 | .NET Framework        | 4.7            | .NETFramework,Version=4.7      | net47                                  | 1.5                   | N/A |
 
-## Supported frameworks
+## Supported target frameworks
 
-A framework is typically referenced by a short TFM. A TFM value can represent one or multiple frameworks. The following table shows the frameworks supported by the .NET Core SDK and the NuGet client. Equivalents are shown within brackets (`[]`).
+A target framework is typically referenced by a compact TFM. A TFM value can represent one or multiple target frameworks. The following table shows the target frameworks supported by the .NET Core SDK and the NuGet client. Equivalents are shown within brackets (`[]`).
```

Comment:
ei ei ei ...... *platforms!* yikes. I meant "target platforms." lol This is assuming, of course, that the .NET Standard topic keeps its language intact on that point (i.e., it doesn't get "platform" converted to "runtime" everywhere; and it goes forward with language such as "Defines uniform set of BCL APIs for all .NET platforms to implement," "all versions of .NET Standard and the platforms supported," "Find the row that indicate the .NET platform you want to run on.," etc.).

I'm pulling the sentence because .NET Standard is covered and linked well above and there's no important reason to mention it prior to the reader looking at the table.